budcamp/Homework3/Homework3/task1/History.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Homework3
+{
+    internal abstract class History
+    {
+        public string DeadRussians { get; set; }
+        public virtual void PolishLithuanianCommonwealth() 
+        {
+            Console.WriteLine("time the moskow has to be burned");
+        }
+
+        public abstract void Symbols();
```

Comment:
Same here, method name should always describe an action, rename to something like `PrintSymbols` or `GetSymbols` for example.